File: School source codes/Lab 7/Sumeven.cs
using System;

class MainClass {
  public static void Main (string[] args) {
    
    Console.Write("Entered Value: ");
    int input = Convert.ToInt32(Console.ReadLine());
    while (input >= 20 && input <= 60)
    {
      int sum = 0;
      int count = 2;
      while (count <= input) 
      {
        sum = sum + count;
        count += 2;
      }
      Console.WriteLine("Sum of even numbers between 2 and " + input + " is " + sum);
      
      Console.Write("Entered Value: ");
      input = Convert.ToInt32(Console.ReadLine());
    }
      Console.WriteLine("Yo that's actually an invalid number for this program. Please try again.");
  }
}

File: School source codes/Lab 7/Sumeven.rb
print "Entered Value: "
input = gets.chomp.to_i
while (input >= 20 && input <= 60)
  sum = 0
  count = 2
  while (count <= input) 
    sum += count;
    count += 2
  end
  puts "Sum of even numbers between 2 and #{input} is #{sum}"
  print "Entered Value: "
  input = gets.chomp.to_i
end
puts "Yo that's actually an invalid number for this program. Please try again."